cmd/neofs-cli/modules/session/create.go

```diff
@@ -65,7 +67,17 @@ func createSession(cmd *cobra.Command, _ []string) {
 	if lfArg, _ := cmd.Flags().GetUint64(commonflags.Lifetime); lfArg != 0 {
 		lifetime = lfArg
 	}
-
+	exp, _ := cmd.Flags().GetUint64(commonflags.ExpireAt)
+	if exp != 0 {
+		var netInfoPrm internalclient.NetworkInfoPrm
+		netInfoPrm.SetClient(c)
+		ni, err := internalclient.NetworkInfo(ctx, netInfoPrm)
+		common.ExitOnErr(cmd, "can't get current epoch: %w", err)
+		currEpoch := ni.NetworkInfo().CurrentEpoch()
+		if exp < currEpoch {
```

Comment:
so we ensure that `lifetime` gonna be negative (underflowed in fact)?

have you considered my primary idea about changing `CreateSession`?  it can look like `CreateSession(ctx context.Context, dst *session.Object, c *client.Client, key ecdsa.PrivateKey, expireAt uint64)` (look at the last arg). it will not have `NetworkInfo` inside then and will look the same as the other code in the CLI: we are reading both flags but `expire-at` is our primary goal. no subtraction code, only additional (for `lifetime` cases)